test/e2e/auth/audit.go

```diff
@@ -112,7 +112,7 @@ func expectAuditLines(f *framework.Framework, expected []auditEvent) {
 	}
 
 	// Fetch the log stream.
-	stream, err := f.ClientSet.Core().RESTClient().Get().AbsPath("/logs/kube-apiserver-audit.log").Stream()
+	stream, err := f.ClientSet.Core().RESTClient().Get().AbsPath("/logs/invalid-log-url.log").Stream()
```

Comment:
@sttts 
Can your relabel it later?
I need to remove this commit first. 
I add this change to debug the e2e test.
And I found this test is skipped in github CI.
Sorry, my mistake.

Reply:
OK now. 
This commit for debug is deleted.